src/d/actor/d_a_obj_mkiek.cpp

```diff
@@ -37,65 +42,211 @@ static dCcD_SrcSph sph_check_src = {
     },
 };
 
+const char daObjMkiek::Act_c::M_arcname[] = "MkieK";
+const char daObjMkiek::Act_c::M_envname[] = "MkieK";
+
+Mtx daObjMkiek::Act_c::M_tmp_mtx;
 
 /* 00000078-00000240       .text CreateHeap__Q210daObjMkiek5Act_cFv */
 BOOL daObjMkiek::Act_c::CreateHeap() {
-    /* Nonmatching */
+    J3DModelData* model_data = (J3DModelData*)dComIfG_getObjectRes(M_arcname, MKIEK_BDL_MKIEK);
+    JUT_ASSERT(0x96, model_data != 0);
```

Comment:
Asserts hide macro names
```suggestion
    JUT_ASSERT(0x96, model_data != NULL);
```